Library/UpdateActivityItemProvider.swift

```diff
@@ -13,25 +13,25 @@ public final class UpdateActivityItemProvider: UIActivityItemProvider {
   }
 
   public override func activityViewController(_ activityViewController: UIActivityViewController,
-                                              itemForActivityType activityType: UIActivityType) -> Any? {
+                                              itemForActivityType activityType: UIActivityType?) -> Any? {
 
     guard let update = self.update else {
       return self.activityViewControllerPlaceholderItem(activityViewController)
     }
 
     switch activityType {
-    case UIActivityType.mail:
+    case UIActivityType.mail?:
```

Comment:
Maybe another `guard` to be sure the `activityType` is not `nil` else return the default string?

Reply:
Good suggestion!